src/Contacts/CNEnums.cs

```diff
@@ -85,6 +85,8 @@ public enum CNErrorCode : nint {
 		CommunicationError = 1,
 		DataAccessError = 2,
 		AuthorizationDenied = 100,
+		[iOS (11,3), Mac (10,13,3)]
```

Comment:
Interesting it's marked as 10.13.3 instead of 10.13.4 which is the new beta. As long as the tests pass though...

Reply:
Yep, I wanted to stick to headers since this particular case there is not much of an issue

Reply:
availability attributes on error codes are not very useful (e.g. introspection can't use it) and we generally ignore them
the reality is that the contract is the (enum) type, not the value(s)

Reply:
Yeah, I still think it is a good thing to have because of IDE help on this, at least it raises awareness on when it was introduced

Reply:
Oh yea didn't realize when reviewing it was an error code. Should be removed IMHO.